subproject-lifecycle.md

```diff
@@ -0,0 +1,82 @@
+# Lifecycle of a subprojects
+
+## Prerequisites
+
+We recognize 2 options how a subproject can be created in OSG. Either this is a preexisting workstream that members want to transfer under SIG governance or it is a brand new workstream which a member thinks that it should be governed by a SIG in OSG.
+
+### Create a new subproject
+
+1. Establish a relation with a SIG that takes patronage over your subproject idea. This SIG will become subproject's owner.
+   - Either get a verbal agreement on a SIG meeting, documented in SIG meeting notes
+   - Or file an issue in open-services-group/community with subproject details and get a SIG lead approval on the issue
+2. Define and declare scope of the subproject and end goal
+
+### Import a workstream to this organization as a subproject
+
+1. Establish a relation with a SIG that takes patronage over your subproject idea. This SIG will become subproject's owner.
+   - Either get a verbal agreement on a SIG meeting, documented in SIG meeting notes
+   - Or file an issue in open-services-group/community with subproject details and get a SIG lead approval on the issue
+2. Communicate the transfer to all the current owners of the workstreams
+3. There must be existing GitHub repository for the workstream
+4. Declare project owners in [OWNERS file](adr/0000-use-OWNERS-rather-than-github-teams.md)
+
+## Process
+
+1. Fork `open-services-group/community`
+2. Checkout a feature branch named after your project
+3. Update [`sigs.yaml`](./sigs.yaml) file with following:
+    1. Locate the SIG entry in `sigs` top level key, it has a property `dir: sig-<NAME>`.
+    2. If your subproject has regular meetings, update the `.meetings` section with:
+
+       ```yaml
+       meetings:
+         - description: "Description containing the subproject name"
+           day: "[Monday,Tuesday,...]"
+           time: "HH:MM"
+           tz: "[UTC|ET (East Time)|CET (Central Europe Time)|...]"
+           frequency: "[weekly|bi-weekly]"
+           url: https://meet.google.com/...
+       ```
+
+    3. Add your subproject to the `.subprojects` section:
+
+       ```yaml
+       subprojects:
+         - name: <NAME>
+           owners:
+             - https://raw.githubusercontent.com/<ORG>/<REPO>/main/OWNERS  # Do not panic if the repository doesn't exist yet
+       ```
+
+4. Update [`github-config.yaml`](./github-config.yaml) file:
+    1. If you are creating a brand new subproject, please initialize a new repository as:
+
+       ```yaml
+       orgs:
+         open-services-group:
+           repos:
+             <REPOSITORY_NAME>:
+               description: Describe the subproject in one sentence.
+               has_projects: false
+               has_wiki: false
+               ...
+       ```
+
+    2. Optionally you can create new GitHub team to maintain this repository:
```

Comment:
I think we should do this (non optional) and describe its relation to the SIG leads? what is the difference?

Reply:
Hm.. what do others think? Should this be mandatory? cc @schwesig @codificat ?

There's no relation to SIG leads, subproject can be maintained by SIG members, no need for hands on participation of SIG leads I think. WDYT?